CMakeLists.txt

```diff
@@ -270,8 +270,6 @@ message("-- ETHSTRATUM       Build Stratum components (experimental)  ${ETHSTRAT
 message("------------------------------------------------------------------------")
 message("")
 
-set(CMAKE_THREAD_LIBS_INIT pthread)
```

Comment:
This is great. I suspect Linux requires this though, so it needs an if (windows) condition.